alibi/confidence/model_linearity.py

```diff
@@ -333,7 +333,7 @@ def _linearity_measure(predict_fn: Callable,
     return score
 
 
-def _infer_feature_range(X_train: np.ndarray) -> np.ndarray:
+def infer_feature_range(X_train: np.ndarray) -> np.ndarray:
```

Comment:
What's the reason for this?

Reply:
`_infer_feature_range` is used within a notebook example [here](https://docs.seldon.io/projects/alibi/en/latest/examples/linearity_measure_fashion_mnist.html?highlight=_infer#Fashion-MNIST-data-set). I've been working under the principle that if it's documented in an example it should be public API.

Reply:
Ah I think I changed it in response to [this](https://github.com/SeldonIO/alibi/pull/583#discussion_r836679555). Although I've since removed it from the `__init__` due to another requested change.